tools/mOTUs/motus_merge.xml

```diff
@@ -0,0 +1,90 @@
+<tool id="mereg_mOTUs_tables" name="Merge" version="0.1.0+galaxy0" profile="21.05">
+    <description>Merge mOTUs aboundance or count tables</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="biotools"/>
+    <expand macro="requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+    #if $db.db_source == 'cached'
+            #set $DATABASE_DIR=$db.db_cached.fields.path
+        #else
+            #set $database_path = $db.database
+            file $database_path &&
+            mkdir database_dir &&
+            tar -xvf '$database_path' -C database_dir/ &&
+            #set $DATABASE_DIR = 'database_dir/db_mOTU' ##the actual DB folder in the archive called db_mOTU
+        #end if
+        echo "Database Directory:" $DATABASE_DIR  &&
+        motus merge
+        -db '$DATABASE_DIR'
+        -i '${','.join([str($file) for $file in $files])}'
+        #if $add_profiles
+        -a '$add_profiles'
+        #end if
+        -o '$output'
+    $biom_format
+    ]]></command>
+    <inputs>
+        <param name="files" type="data" format="txt" multiple="true" label="mOTU profiles to merge" />
```

Comment:
can this also be biom files?

Is there any advantages of using the txt format instead of the BIOM format? I'm asking because we could simply drop the txt format and not support it - forcing the using to use BIOM a standard.